src/app/api/routing.py

```diff
@@ -4,12 +4,14 @@
 from fastapi import HTTPException, Path
 
 from app.api import crud, security
-from app.api.schemas import UserAuth, UserCreation
+from app.api.schemas import UserAuth, UserCreation, UserRead
+from app.api.schemas import AccessCreation, AccessRead, AccessAuth
+from app.api.schemas import DeviceAuth, DeviceCreation, DeviceOut
+from app.db import access as access_table
```

Comment:
I think we can avoid this import by passing the `access_table` argument of `create_access` down to `_create_access` :)

Reply:
_create_access is called by create_user and create_device as well and I'd rather not add the access_table as input of those functions in the user and devices routes

Reply:
I was thinking it would actually be helpful because, from the routing code, you know straight away which tables will be accessed by any route. But I understand your point, happy to move forward with this :+1: